src/iterated_integrals.jl

```diff
@@ -0,0 +1,130 @@
+# This function calculates WikJ, a mxm Array for a m dimensional general noise problem,
+# which is a approximation to the second order iterated integrals
+# this is implementation of the section 4 of the paper doi:10.1016/j.cam.2006.05.037
+
+function get_iterated_I!(integrator, cache::StochasticDiffEqConstantCache)
+    @unpack dt, u, uprev, t, p, W = integrator
+    @unpack m_seq, WikJ = cache
+
+    dW     = W.dW
+    m      = length(dW)
+    M      = m*(m-1)/2
+
+    if typef(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+        cache.WikJ = 1//2 .* J .^ 2
+    else
```

Comment:
what about the commutative noise case? That should be in the RKMilCommute cache constructor already.

Reply:
We don't have a function for commutative noise like we have for diagonal so I thought let's consider it in general case. Otherwise, It would be difficult maintaining a commutative noise version of every solver.

Reply:
That's what this is: https://github.com/JuliaDiffEq/StochasticDiffEq.jl/blob/master/src/perform_step/low_order.jl#L198-L202 . It should just call to a common routine in here to fill `WikJ` instead, and if it's commutative that's the calculation.